randovania/games/am2r/json_data/Genetics Laboratory.txt

```diff
@@ -674,20 +674,39 @@ Extra - minimap_data: [{'x': 7, 'y': 20}, {'x': 8, 'y': 20}, {'x': 9, 'y': 20}]
   * Back Only to Queen Arena Access/Dock to Queen Arena
   > Event - Queen
       All of the following:
+          All of the following:
+              # First Phase
+              Any of the following:
+                  # Weapon Requirements
+                  Combat (Expert) and Can Use Bombs
+                  Combat (Intermediate) and Can Use Charged Bomb Jump
+                  All of the following:
+                      Ice Beam
+                      Missiles or Super Missiles
+                      Any of the following:
```

Comment:
Can this get a comment with "defend against blobs"?
Seeing this now also makes me wonder whether we should have *some* ammo reqs for trickless, to reduce the farming.